docs/modules/ROOT/nav.adoc

```diff
@@ -2,3 +2,7 @@
 
 * xref:access-control.adoc[Access Control]
 * xref:crypto.adoc[Cryptography]
+
+* xref:tokens.adoc[Tokens]
+** xref:erc20.adoc[ERC-20]
+** xref:erc721.adoc[ERC-721]
```

Comment:
IMHO `erc721` should not be a part of this PR.

Reply:
Basically there is a text like "You've probably heard of the ERC-20 or ERC-721 token standards, and that's why you're here. Head to our specialized guides to learn more about these:" similar to the other OZ documentation. Since we are already mentioning erc-721 will not be such a great difference. It will be added soon also. Wdt?

Reply:
But yeah, removed it